the-experts-fe/src/components/contractorForm/ContractorForm.jsx

```diff
@@ -1,7 +1,7 @@
 import React, { useState, useEffect } from "react";
-import "./registerationForm.css";
+import "../../../public/css/form.css";
```

Comment:
Why have you moved your `form.css` so far away in public folder? Can't it be in the same folder as `ContractorForm.jsx`?

Reply:
because i used this style in several forms.
i didnt wanted to duplicate this file in each component form folder